docs/tutorials/fsi-montecarlo-on-batch/README.md

```diff
@@ -0,0 +1,201 @@
+
+Monte Carlo simulation is a mathematical technique that can be used to estimate the [value at risk (VaR) of a financial portfolio](https://en.wikipedia.org/wiki/Value_at_risk). The basic idea is to generate a large number of random price paths for the assets in the portfolio, and then calculate the VaR as the minimum loss that would be expected to occur with a certain probability.
+
+1. Choose a time horizon and a confidence level.
+1. Generate a large number of random price paths (random walks) for the assets in the portfolio.
+1. Calculate the portfolio value at the end of each price path.
+1. Determine the VaR as the minimum portfolio value that occurs with the specified confidence level.
+
+
+## Tutorial architecture
+The overall structure of this tutorial is as follows:
+
+* The Monte Carlo simulation is managed with [Batch](https://cloud.google.com/batch/docs/get-started). 
+* The output of the Monte Carlo simulation is published to a [PubSub](https://cloud.google.com/pubsub/docs/overview) topic.
+* The PubSub data is entered into [BigQuery](https://cloud.google.com/bigquery) 
+via a [PubSub BigQuery subscription](https://cloud.google.com/pubsub/docs/bigquery)
+* The data is visualized via a [Vertex AI Workbench](https://cloud.google.com/vertex-ai-workbench) 
+[Jupyter Notebook](https://jupyter.org/) 
+
+<img src="./images/architecture.png" width="800" />
+
+## Basic getting started
+
+1. Get the HPC Toolkit configured.
+
+Build the `ghpc` binary:
+
+```shell
+git clone https://github.com/GoogleCloudPlatform/hpc-toolkit
+cd hpc-toolkit
+make
+./ghpc --version
+./ghpc --help
+```
+
+2\. Enable Google Cloud Resource Manager:
+
+```
+gcloud services enable cloudresourcemanager.googleapis.com
+```
+
+3\. Run `ghpc` on the blueprint `fsi-montecarlo-on-batch.yaml`
+
+```
+./ghpc create community/examples/fsi-montecarlo-on-batch.yaml
+```
+
+If successful, you will see output similar to:
+```
+To deploy your infrastructure please run:
+
+./ghpc deploy fsimontecarlo
+```
+4\. Edit the file to update `<my_project>` with your project_id:
+
+```
+  project_id: <my_project>
+```
+
+5\. Deploy the blueprint as instructed:
+
+```
+./ghpc deploy fsimontecarlo
+```
+If successful, this will prompt you:
+```
+Summary of proposed changes: Plan: 22 to add, 0 to change, 0 to destroy.
+(D)isplay full proposed changes,
+(A)pply proposed changes,
+(S)top and exit,
+(C)ontinue without applying
+Please select an option [d,a,s,c]:
+```
+Respond with `apply`, "a". You may be asked to respond twice. 
+
+When the job is complete it will indicate `Succeeded`. You can then proceed to the next section.
+
+At this point, all the required infrastructure has been deployed.
+
+## Open Vertex AI Workbench
+
+1. Go to the Vertex AI Workbench Notebooks instances in the Google Cloud Console:
+```
+https://console.cloud.google.com/vertex-ai/workbench/user-managed
```

Comment:
nit: just make this a link instead of a code block.

Reply:
My concern here, is that the user may be reading in one browser profile and executing in another, at least that is my typical experience. Being able to copy the link makes more sense in that context. Popping a window into your browsing profile is not helpful.

Reply:
Done